app/controllers/application_controller.rb

```diff
@@ -2,4 +2,50 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  helper_method :current_user, :current_order, :current_user_is_owner, :require_login
+
+  def current_user
+      @user ||= User.find_by(id: session[:user_id])
+  end
+
+  def current_user_is_owner
```

Comment:
With methods that check whether something is true or false it is customary to end it with a question mark. So

`def current_user_is_owner?`